src/components/Pokedex/evolutionGraph.js

```diff
@@ -1,35 +1,38 @@
 import * as React from 'react';
 import { Typography } from '@mui/material';
 import { useState } from 'react';
-import {getEvolutionData} from '../../../dexUtils';
-
+import { getEvolutionData } from '../../../dexUtils';
 
 export default function EvolutionGraph(props) {
-    const {dexId} = props;
-    return (
-        <div className='container'>
-            <div className='row' style={{margin: 'auto', textAlign: 'center'}}>
-            <span className='col col-12'>
-                <Typography variant='h6' sx={{margin: 'auto'}}>Evolutions</Typography>
-            </span>
+  const { dexId } = props;
+  return (
+    <div className='container'>
+      <div className='row' style={{ margin: 'auto', textAlign: 'center' }}>
+        <span className='col col-12'>
+          <Typography variant='h6' sx={{ margin: 'auto' }}>
+            Evolutions
+          </Typography>
+        </span>
+      </div>
```

Comment:
I have only changed this one single file as demonstration purpose - we will run into merge conflicts if I touch every file, so we should merge this PR and then rebase existing branches based on Pokedex